src/lib-voyager.tsx

```diff
@@ -220,11 +231,35 @@ export class Voyager {
 /**
  * Create an instance of the voyager application.
  *
- * @param {Container} container css selector or HTMLElement that will be the parent
- *                              element of the application
- * @param {Object}    config    configuration options
- * @param {Array}     data      data object. Can be a string or an array of objects.
+ * @param {Container}       container css selector or HTMLElement that will be the parent
+ *                                    element of the application
+ * @param {VoyagerParams}   params    Voyager params. {data, config}. Any new additional parameter
+ *                                    required by voyager can be added here as named parameter for easier visibilty
  */
-export function CreateVoyager(container: Container, config: VoyagerConfig, data: Data): Voyager {
-  return new Voyager(container, config, data);
+export function createVoyager(container: Container, params: VoyagerParams): Voyager {
+  return new Voyager(container, params);
+}
+
+
+/**
+ * Create an instance of the voyager application.
+ *
+ * @param {Container}       container css selector or HTMLElement that will be the parent
+ *                                    element of the application
+ * @param {Object}          config    configuration options
+ * @param {Array}           data      data object. Can be a string or an array of objects.
+ */
+export function CreateVoyager(container: Container, data: Data, config: VoyagerConfig): Voyager {
```

Comment:
We should not offer two versions of a function with a capital C and a lowercase C.  

So you have two options:

a) Remove `CreateVoyager`.  (Yes, this will break things, but we are currently under pre-release version, so better make things right right now. I'll note in the release note that the next release includes this breaking changes.)  

b) Do not add `createVoyager` with the new parameter since this PR only modifies VoyagerConfig and doesn't really add new parameter to `CreateVoyager` anyway.  

I strongly prefer a) since it's better prepared for the future.  (We also previously let `CreateVoyager` with capital `C` slips in by mistake -- so better fix it before it is too late.)

If you do a), please also help:

1) Update README.md accordingly
2) submit a relevant PR for `voyager-electron`to update relevant API calls

Reply:
@kanitw ok makes sense. I added it as a separate function for gradually letting users know that this function will be deprecated in the next release. But since voyager is still in alpha/beta releases I guess it should be ok. Its a simple API. Will revert back.

Reply:
> Will revert back.

Note that if you do b), we will probably still do a) in the future anyway and you'll have to update the API calls.  Maybe it's better if you help us do a) now :p

Reply:
👍 Will do a). Regarding PR for voyager-electron: There is an already opened PR. Not sure if I can create another one. But will see if I can spare sometime this weekend for it :)